setup.py

```diff
@@ -58,6 +58,11 @@ def read_file(filename, encoding='utf8'):
 else:
     install_requires.append('mock')
 
+if sys.version_info < (2, 7, 9):
```

Comment:
Actually I added them because they were missing.

Since requests is a dependency here and that it is used to call HTTPS URLs we need to add this in order to use a secure communication (and remove the InvalidPlatformWarning)